docs/synapse/userguides/storm_adv_vars.ipynb

```diff
@@ -1100,6 +1091,15 @@
     "# Close cortex because done\n",
     "_ = await core.fini()"
    ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. _Libraries: ../autodocs/stormtypes_libs.html\n",
+    ".. _Node: ../autodocs/stormtypes_prims.html#node\n",
```

Comment:
Do we not have automatically generated header values you can link to by reference for these?